### ISSUE TITLE
Color Background at Widget should be possibel, direkt, but not with a Lua

### ISSUE THREAD

**Issue by hrenz** [FrSkyRC/ETHOS-Feedback-Community#4028]
Would be nice to select some colours 
at Text Widget Background or Flightphases, or a Bidmap

Red, Green, Yellow, Blue, (not more) 

Example
![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/6217896/643e5395-e11c-49ce-90c6-7bcd0156b1b0)

**Comment by philandroid36** [comment]
Yes. For me, background is necessary to make widgets transparent
